xchange-binance/src/main/java/org/knowm/xchange/binance/service/BinanceAccountService.java

```diff
@@ -287,6 +304,52 @@ public TradeHistoryParams createFundingHistoryParams() {
                 });
       }
 
+      final String finalEmail = email;
```

Comment:
I don't think this is needed

Reply:
Lambda need final field. email field is null then affected to a value if needed.